nemoguardrails/llm/prompts/cohere-commandr.yml

```diff
@@ -0,0 +1,158 @@
+# Prompts for Cohere Command-r.
+prompts:
+  - task: generate_user_intent
+    models:
+      - cohere/command-r
+    messages:
+      - type: system
+        content: "{{ general_instructions }}"
+
+      - type: system
+        content: |-
+          You are a helpful ai assistant. Your response should include three parts: User intent, Bot intent and Bot message.
+          - User intent: <Extract user intent>
+          - Bot intent: <Show what you think>
+          - Bot message: <Actual response>
+          Each part can only appear once.
+
+      - "{{ sample_conversation | to_messages }}"
+
+      - type: system
+        content: "Answer user question with following part: User intent"
+
+      - "{{ examples | to_messages}}"
+
+      - type: system
+        content: "Answer user question with following parts: User intent, Bot intent and Bot message."
+
+      - "{{ sample_conversation | first_turns(2) | to_messages }}"
+      - "{{ history | colang | to_messages }}"
+
+    output_parser: "user_intent"
+
+  - task: generate_user_intent
```

Comment:
This appears twice with identical prompts. 
Did you also want to include the `generate_intent_steps_message` task to work with a single LLM call or this is an error?

Reply:
The second one was supposed to be the compact variant. Since they are identical, removing the second one.